src/applications/disability-benefits/all-claims/tests/fixtures/data/transformed-data/maximal-toxic-exposure-test.json

```diff
@@ -0,0 +1,181 @@
+{
+  "form526": {
+    "standardClaim": false,
+    "isVaEmployee": true,
+    "homelessOrAtRisk": "homeless",
+    "homelessHousingSituation": "other",
+    "otherHomelessHousing": "Under a bridge",
+    "needToLeaveHousing": true,
+    "homelessnessContact": {
+      "name": "Name Here and invalid chars with email elmer fud com",
+      "phoneNumber": "1231231234"
+    },
+    "bankAccountType": "Checking",
+    "bankAccountNumber": "1233",
+    "bankRoutingNumber": "123123123",
+    "bankName": "Bigbank Co.",
+    "phoneAndEmail": {
+      "primaryPhone": "4445551212",
+      "emailAddress": "test2@test1.net"
+    },
+    "mailingAddress": {
+      "country": "USA",
+      "addressLine1": "123 Main",
+      "city": "Bigcity",
+      "state": "AK",
+      "zipCode": "12345"
+    },
+    "forwardingAddress": {
+      "country": "USA",
+      "addressLine1": "321 Secondary",
+      "city": "Smallcity",
+      "state": "NY",
+      "zipCode": "54321",
+      "effectiveDate": { "from": "2099-12-01", "to": "2100-01-01" }
+    },
+    "toxicExposureConditions": {
+      "asthma": true
+    },
+    "gulfWar1990": {
+      "airspace": true
+    },
+    "gulfWar1990Locations": {
+      "afghanistan": {
+        "startDate": "1990-01-01",
+        "endDate": "1990-12-31"
+      }
+    },
```

Comment:
This file was copied from `src/applications/disability-benefits/all-claims/tests/fixtures/data/transformed-data/maximal-test.json` and lines 36 - 47 added for TE